src/libs/routes/errorHandler.ts

```diff
@@ -1,11 +1,11 @@
 import { Request, Response, NextFunction } from 'express';
 
-export default function errorHandler (err, req: Request, res: Response, next: NextFunction ) {
+export default function errorHandler(err, req: Request, res: Response, next: NextFunction) {
     res.json({
         error: err.error,
         status: err.code,
-        message: err.message || "Error",
+        message: err.message || 'Error',
         timestamp: new Date()
-        
-    })
-}
\ No newline at end of file
+    });
+}
+
```

Comment:
use destructing and handle undefined value cases

Reply:
Used destructing and handle all undefined value cases